### PR TITLE
fix(cordova/apple/ios): update Code Signing Identity

### PR DESCRIPTION
**Pull request by daniellacosse**
I... really don't know if this is the correct way to do this, but there seems to be a code signing identity mismatch between what's in our certs and what's in the xcode project. I've... updated that via XCode. And it was able to build. But it feels incorrect.

### CODE REVIEW

File: src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -373,9 +373,7 @@
 				LastUpgradeCheck = 1010;
 				TargetAttributes = {
 					1D6058900D05DD3D006BFB54 = {
-						DevelopmentTeam = QT8Z3Q9V3A;
```
Comment (daniellacosse) [inline src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj:376]: It's weird that the team disappeared. Should I attempt to manually put that back? Should it be in our repo at all??

File: src/cordova/apple/xcode/ios/Outline.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fortuna) [inline src/cordova/apple/xcode/ios/Outline.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: Why are you adding this file?

**Review by fortuna — COMMENTED** [review]
Can you show me what's changing in our meeting?

File: src/cordova/apple/xcode/ios/Outline.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 2 above)
Comment (daniellacosse) [reply src/cordova/apple/xcode/ios/Outline.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: I don't know

**Pull request closed without merging**